README.md

```diff
@@ -203,6 +203,14 @@ EOF
 ```
 
 
+## Presentations on CDI
+
+See the following presentations for more information on CDI:
+
+* Kubecon EU 2023 - Node Resource Management: The Big Picture - https://sched.co/1HyVB - https://youtu.be/LjhgklNNAtA
+* Kubecon NA 2023 – The hidden heroes behind AI: Making sense of GPUs and TPUs in k8s - https://sched.co/1R2sf - https://youtu.be/H5gN8fHn3fo
+* Kubecon EU 2024 – Sharing is Caring: GPU Sharing and CDI in Device Plugins  – https://sched.co/1YeQ7 – https://youtu.be/Q2GuTUO170w
+
```

Comment:
Some older ones to include:
Kubecon NA'20: 
Panel: Introduction to the Container Orchestrated Device (COD) Working Group - Renaud Gaubert, NVIDIA; Mike Brown, IBM; Mrunal Patel & Urvashi Mohnani, Red Hat; & Alexander Kanevskiy, Intel, 
https://sched.co/ekDv  , https://www.youtube.com/watch?v=1k1sdf4mrpI

Kubecon NA'22, Improving User Experience For Device Consumption In Kubernetes - Alexander Kanevskiy & Patrick Ohly, Intel & Kate Goldenring, Fermyon
https://sched.co/182GN , https://www.youtube.com/watch?v=0Gnh86queVQ